Exercise_5.java

```diff
@@ -1,20 +1,80 @@
-class IterativeQuickSort { 
-    void swap(int arr[], int i, int j) 
-    { 
-	//Try swapping without extra variable 
-    } 
-  
-    /* This function is same in both iterative and 
+class IterativeQuickSort {
+    /**
+     * Time Complexity: O(n * log(n))
+     * Space Complexity: O(1) in-place sorting
+     * Any problem you faced while coding this : I haven't understood the left and right partition being added to stack,
+     * why stack is being made empty after the while loop, why stack is being pushed with low and high value.
+     * As well the swap without additional variable works fine but in code it's providing irrelevant results.

```

Comment:
Here you are performing the same steps that would happen in a recursion but with the help of a `stack` .